solr/core/src/test/org/apache/solr/servlet/DirectSolrConnectionTest.java

```diff
@@ -22,6 +22,7 @@
 import org.apache.solr.common.params.CommonParams;
 import org.junit.BeforeClass;
 
+@Deprecated
```

Comment:
[Q] Why are we deprecating this?  Is this related to the "version" removal?

Reply:
Totally unrelated to this PR; sorry.  DirectSolrConnection is deprecated so I thought deprecating its test to would be nice even though totally unnecessary.